data-model/src/main/scala/za/co/absa/enceladus/model/MappingTable.scala

```diff
@@ -76,4 +80,21 @@ case class MappingTable(name: String,
           AuditFieldName("schemaVersion", "Schema Version"))) ++
         super.getSeqFieldsAudit(newRecord, AuditFieldName("defaultMappingValue", "Default Mapping Value")))
   }
+
+  override def exportItem(): String = {
+    val mapper = new ObjectMapper()
+    mapper.registerModule(DefaultScalaModule)
+
+    val root: ObjectNode = mapper.createObjectNode()
+    val defaultMappingValueJsonList: ArrayNode = mapper.valueToTree(defaultMappingValue.toArray)
+
+    root.put("name", name)
```

Comment:
Since you are already using the jackson object mapper, is there a reason why you do the marshalling manually instead of  using Jackson's `toJson()` methods (while marking some of the fields you want to omit from marshalling by the means of `@JsonIgnoreProperties` and similar)?

Hinting something along the lines of [1](https://coderwall.com/p/o--apg/easy-json-un-marshalling-in-scala-with-jackson) and [2](https://stackoverflow.com/questions/25654811/jsonignore-serialising-scala-case-class-property-using-jackon-and-json4s)

Reply:
If I am not wrong we use full (all attribute) JSON SerDe as it is tested here [JsonSerializerSuite](https://github.com/AbsaOSS/enceladus/blob/develop/dao/src/test/scala/za/co/absa/enceladus/dao/rest/JsonSerializerSuite.scala) for other (presumably mongo related) purposes. 

Now how I understood the ignore, it is applied `case class` wide and cannot be specified specially for one method.

Reply:
Yes, that makes sense. That is the point I was missing. 👍